docs/schema-loader.md

```diff
@@ -446,6 +446,8 @@ However, the following types in JDBC databases are converted differently when th
 | TEXT     | VARCHAR(64)   | VARCHAR(10485760) | VARCHAR2(64) |
 | BLOB     | VARBINARY(64) |                   | RAW(64)      |
 
+Also, `BIGINT` of Scalar DB supports a value from -2^53 to 2^53 even if you use any databases.
```

Comment:
Looks good.

Just another option: I think you can simply say "the value range of BIGINT is ..." without the term `supports` like
```
The value range of BIGINT in Scalar DB is from -2^53 to 2^53 regardless the underlying database.
```

Reply:
I see. Thank you for another option. I will keep this in mind for future reference.

It is difficult to simply and accurately describe information in a document...

Reply:
@kota2and3kan 
The one from @komamitsu sounds a bit more right in English (except for the missing `of`), so it would be great if you can update it as follows. (`even if` part sounds a bit weird to me, but English is difficult...)

```
The value range of BIGINT in Scalar DB is from -2^53 to 2^53 regardless of the underlying database.
```

Reply:
@feeblefakie  
Thank you for your review!
I updated it. Please take a look again!